ProjectSidewalk/SidewalkWebpage
# added updated text to label map

Resolves #3017 

Added a helper method to help update just the validation row on the front end when a user clicks one of the three buttons for validating labels on the label map. 

##### Before/After screenshots (if applicable)

Before:

Refer to #3017 for original video

After:


https://user-images.githubusercontent.com/56641275/230795813-d7f9f897-e313-4bfb-9335-38f59352cc9b.mov


Funny bug but doesn't affect actual count


https://user-images.githubusercontent.com/56641275/230795874-4ad7f949-30b9-433c-8051-bc65d48d9216.mov


##### Testing instructions
1. Go to LabelMap
2. Select a label and choose "Agree", "Disagree", or "Unsure"
3. Observe validation count go up for button chosen
4. Exit label and re-open to observe accurate validation count (based off last one chosen)

##### Things to check before submitting the PR
